integration-tests/tests/aws/post-ivq-answers/post-ivq-answers-happy.test.ts

```diff
@@ -0,0 +1,105 @@
+import { stackOutputs } from "../resources/cloudformation-helper";
+import { clearItems, populateTable } from "../resources/dynamodb-helper";
+import { executeStepFunction } from "../resources/stepfunction-helper";
+
+describe("post-ivq-answers-happy", () => {
```

Comment:
no need for `-happy` in this case because this file contains all the tests, same with the file name